### PR TITLE
Add Help request model

### PR DESCRIPTION
**Pull request by 98ZhaoJeffrey**
## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/7e51fde5f22d448b8226f2cfa4dd93d5?v=b2dc0dba06574651a8b6063bdd0d74e0&p=a82d483d0f564e0bb4104c4acc40c9f4&pm=s)

## Implementation description
* Implemented a new model for getting help with content
* 

## Steps to test
1. Look at the file "test_help_model.ts" and make any changes needed
2. Run it with ts-node ./test_help_model.ts in the backend folder
3. Check if the model exists in MongoDB

## What should reviewers focus on?
* Focus on the course help request portion, the facilitator and learner aspect is from Cherry's ticket and may change
* I stored the learner and facilitator as the object since they only container a little bit of extra info,
* For the unit, module and page, I stored them as the object id instead. I think this is better since if were to store them as the objects, we would have lots of duplicate information stored since CourseUnits include all Modules which includes all Pages and we only need to know the location of each individual part.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by achandrabalan — APPROVED** [review]
LGTM !

**Review by ji-mmyliu — APPROVED** [review]
LGTM with quick linting fixes. Love the unit tests!!

**Review by carolynzhang18 — COMMENTED** [review]
Looks pretty good!! A couple of suggestions - lemme know what you think! 🧠

P.S. might wanna pull in the latest main, there are some newer changes on the user model now :  )

File: backend/models/helprequest.mgmodel.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+} from "./user.mgmodel";
+
+export interface HelpRequest extends Document {
+  id: string;
+  message: string;
+  learner: typeof Learner;
+  facilitator: typeof Facilitator;
+  unit: string;
+  module: string;
+  page: string;
```
[the remark this answers is not in the document]
Comment (98ZhaoJeffrey) [reply backend/models/helprequest.mgmodel.ts]: read my slack msg :D

**Review by carolynzhang18 — APPROVED** [review]
lookin good!! we can take away the `test_help_model` file and good to go 😎

**Pull request merged**